### DISCUSSION TITLE
Shadcn Integration

### DISCUSSION THREAD

**Discussion by JaredDevine** [alex8088/electron-vite#551] [Q&A] [answered]
Hello,

I'm trying to get shadcn installed with my electron-vite project. I've been struggling for hours trying to get the paths to resolve and fix the typescript errors. Any help is greatly appreciated. I've been using the shadcn Vite docs as they are the closest to the electron setup:

https://ui.shadcn.com/docs/installation/vite

**Comment by LamidoKing** [comment, marked as answer]
I am using it with shadcn without any problems, but don't expect the shadcn CLI to work with this setup because the CLI expects vite.config.ts, while this project uses electron.vite.config.ts. You can check out this [repository](https://github.com/LamidoKing/electron-vite-shadcn) I made to demonstrate how to use electron-vite with shadcn.

**Reply by JaredDevine** [reply]
Thanks LamidoKing. Yeah, I was trying to get the CLI to work and it was generating the components in the wrong place. I will try your approach.

All the best.
